app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,11 @@
+from abc import ABC
+from app.player import Player
+
+
+class Dwarf(Player, ABC):

```

Comment:
The `Dwarf` class should implement the abstract methods `get_rating` and `player_info` from the `Player` class to be a concrete class. Consider adding implementations for these methods.